### PR TITLE
LPD-22040 Usage of default permissions

### PR DESCRIPTION
**Pull request by smottal**
https://liferay.atlassian.net/browse/LPD-22040

### CODE REVIEW

**Comment by smottal** [comment]
ci:test:um

File: portal-web/docroot/html/taglib/ui/input_permissions/horizontal.jspf [hunk 1]
```diff
@@ -39,6 +39,12 @@ ServiceContext#deriveDefaultPermissions(long, String).
 		<%
 		for (String roleName : roleNames) {
 			Role role = RoleLocalServiceUtil.getRole(themeDisplay.getCompanyId(), roleName);
+
+			List<String> roleDefaultPermissions = Collections.emptyList();
+
+			if ((defaultPermissions != null) && (defaultPermissions.get(role.getName()) != null)) {
```
Comment (andrea-ale-sbarra) [inline portal-web/docroot/html/taglib/ui/input_permissions/horizontal.jspf:45]: roleName

**Review by andrea-ale-sbarra — CHANGES_REQUESTED** [review]
LGTM
Thank you!

**Comment by smottal** [comment]
ci:test:sf

**Comment by smottal** [comment]
ci:test:relevant

**Comment by smottal** [comment]
ci:report:2567253529

**Comment by smottal** [comment]
ci:forward:force

**Pull request closed without merging**